test/CreatorToken.t.sol

```diff
@@ -85,6 +86,7 @@ abstract contract CreatorTokenTest is Test {
     vm.startPrank(_buyer);
     payToken.approve(address(creatorToken), type(uint256).max);
     creatorToken.buy(_totalPrice);
+    vm.warp(block.timestamp + MIN_HOLDING_TIME);
```

Comment:
nit: move the `warp` call outside the `prank`

Reply:
Moved it outside `prank` in the same commit as above: [77d2ad4](https://github.com/showtime-xyz/creator-tokens/pull/38/commits/77d2ad4397a0e6a3979a7554cb64398d9a505ccf)